### PR TITLE
[TwigBundle] use ::class instead path 

### PR DESCRIPTION
**Pull request by BASAKSemih**
| Q             | A
| ------------- | ---
| Branch?       | 6.3
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

**Comment by BASAKSemih** [comment]
There is no other file to modify in this case in TwigBundle

File: src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -135,7 +136,7 @@ private function addTwigOptions(ArrayNodeDefinition $rootNode)
                 ->end()
                 ->scalarNode('autoescape_service')->defaultNull()->end()
                 ->scalarNode('autoescape_service_method')->defaultNull()->end()
-                ->scalarNode('base_template_class')->example('Twig\Template')->cannotBeEmpty()->end()
+                ->scalarNode('base_template_class')->example(Template::class)->cannotBeEmpty()->end()
```
Comment (fabpot) [inline src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php]: I would revert this one as this is an example, not a real class.

**Review by fabpot — CHANGES_REQUESTED** [review]
Can you do it for all bundles for instance if doing it across the board would be too much work? We prefer merging large PRs for this kind of change instead of many small ones.

**Comment by BASAKSemih** [comment]
Yes, I can do it
I'll do one commit for one/bundle.

**Comment by stof** [comment]
Note that you should also reword your PR. `Twig\Template` is not a path.

**Comment by BASAKSemih** [comment]
Do you prefer all here for::class instead, or should we merge this and I do one PR for all other bundles ? (stof i see don't worry ty)

File: src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php (hunk 1 above)
Comment (GromNaN) [reply src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php]: Actually, a class name is expected here and the value of `Template::class === 'Twig\Template'`. I would make the change.

**Pull request closed without merging**